docs/DeveloperGuide.md

```diff
@@ -771,25 +799,44 @@ testers are expected to do more *exploratory* testing.
 
 ### 11.2. Adding a student
 
-1. Deleting a student while all students are being shown
 
-  1. Prerequisites: List all students using the `list` command. Multiple students in the list.
+{ end of `manual_testing#adding_a_student` written by: __________ }
 
-  1. Test case: `delete 1`<br>
-     Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+{ start of `manual_testing#deleting_a_student` written by: Marcus Tan Wei }
 
-  1. Test case: `delete 0`<br>
-     Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
+### 11.3. Deleting a student
 
-  1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-     Expected: Similar to previous.
+1. Deleting a student while all students are being shown
 
-1. _{ more test cases …​ }_
+  1. Prerequisites: List all students using the `liststu` command. Multiple (but less than 100) students in the list.
 
-{ end of `manual_testing#adding_a_student` written by: __________ }
+  1. Test case: `deletestu 1`<br>
+     Expected: First contact is deleted from the list. Details of the deleted contact shown in the ResultDisplay.
 
-{ start of `manual_testing#deleting_a_student` written by: __________ }
+  1. Test case: `deletestu 0`<br>
+     Expected: No student is deleted. Error details shown in the ResultDisplay. 
 
-### 11.3. Deleting a student
+  1. Test case: `deletestu 101`<br>
+     Expected: No student is deleted. Error details shown in the ResultDisplay.
+     
+  1. Other incorrect delete commands to try: `deletestu`, `deletestu x`, `...` (where x is larger than the list size)<br>
+     Expected: Similar to previous.
 
-{ end of `manual_testing#deleting_a_student` written by: __________ }
+1. Deleting a student while only some students are being shown
+   
+   1. Prerequisites: List some students using `findstu` command. Multiple (but less than 100) students are in the list.
+   One or more (but less than 100) student is shown in the StudentListPanel.
+   
+   1. Test case: `deletestu 1`<br>
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the ResultDisplay.
+   
+   1. Test case: `deletestu 0`<br>
+      Expected: No student is deleted. Error details shown in the ResultDisplay. 
+   
+   1. Test case: `deletestu 101`<br>
+      Expected: No student is deleted. Error details shown in the ResultDisplay.
```

Comment:
Perhaps this can be "Similar to previous."?

Reply:
Should I? I'm afraid it wouldn't be clear enough which previous I am referring to.

Reply:
I see. I think this should be fine as well 👍